src/components/Friends/AnonFriendCard.tsx

```diff
@@ -96,11 +104,21 @@ export const AnonFriendCard = ({ friend }: { friend: GenericUser }) => {
             <IconButton aria-label="invite" size="small">
               <EmailOutlinedIcon fontSize="inherit" />
             </IconButton>
-            <IconButton aria-label="delete" size="small">
+            <IconButton
+              aria-label="delete"
```

Comment:
I'm missing some disabling during the time when the request is being sent. We are now running the server locally, but in reality this request can take more than a second